src/integrations/GA4/utils.js

```diff
@@ -173,6 +173,8 @@ function getDestinationEventProperties(
     props,
     contextOp
   );
+  // overriding js sdk page param to capture full url
+  propsWithCustomFields.page_location = window.location.href;
```

Comment:
Can we not update in the mapping part?

Reply:
@utsabc this is basically coming from analytics page call so we need to override it here

Reply:
But where are we mapping in `propsWithCustomFields` prior to this?

Reply:
updated as we discussed